less/all/layout.less

```diff
@@ -95,6 +95,19 @@ body {
     font-size:10px;
 }
 
+#trade a:hover {
+    cursor: pointer;
+}
```

Comment:
Ich würde das hier vllt lieber in den #trade-Block in der `less/all/trade.less` packen